cmds/ocm/commands/ocmcmds/common/handlers/vershdlr/sort.go

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Open Component Model contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package vershdlr
+
+import (
+	"strings"
+
+	"github.com/open-component-model/ocm/cmds/ocm/pkg/processing"
+)
+
+func Compare(a, b interface{}) int {
+	aa := a.(*Object)
+	ab := b.(*Object)
+
+	c := strings.Compare(aa.Component, ab.Component)
+	if c != 0 {
+		return c
+	}
+	return strings.Compare(aa.Version, ab.Version)
```

Comment:
I guess we don't have a real semver-version-comparison available?
String comparison for things like 0.9.2 vs. 0.10.3 will result in 'wrong' order, right?